### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Marklord-dev** [Cyfrin/foundry-full-course-cu#2034] [Q&A]
I am getting the error "Failed to send transaction" when i tried deploying my smart contract to a sepolia testnet using foundry and i have 0.057 sepolia eth in my metamask wallet. Please help
![insufficient funds](https://github.com/user-attachments/assets/fd642efe-182a-4491-8328-d5ba149672b3)

**Comment by EngrPips** [comment]
Hello @Marklord-dev, add `--legacy` to the end of your command and retry. let's see how that plays out.

**Comment by Marklord-dev** [comment]
it worked. Thank you

**Reply by EngrPips** [reply]
Awesome, I am glad you figured it out. Keep crushing it.
